projects/plugins/jetpack/modules/publicize/publicize-jetpack.php

```diff
@@ -274,9 +274,9 @@ public function get_connection_meta( $connection ) {
 	 * Show error on settings page if applicable.
 	 */
 	public function admin_page_load() {
-		$action = sanitize_text_field( wp_unslash( $_GET['action'] ) ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$action = isset( $_GET['action'] ) ? sanitize_text_field( wp_unslash( $_GET['action'] ) ) : null; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
If we defaulted to an empty string, we wouldn't have to check for `empty()` below. Wouldn't that be better?

Reply:
I already removed the `empty()` check below. If it's not set, `$action` will be `null` and `'error' === $action` will always be false, no? I'm happy to change it to an empty string but I don't think it functionally changes anything.

Reply:
Ah right, my bad, I misread. Look good indeed!